sections/exchange/FooterCard/TradeSummaryCard/TradeSummaryCard.tsx

```diff
@@ -32,87 +31,88 @@ type TradeSummaryCardProps = {
 	isApproved?: boolean;
 };
 
-const TradeSummaryCard: FC<TradeSummaryCardProps> = ({
-	submissionDisabledReason,
-	onSubmit,
-	feeReclaimPeriodInSeconds,
-	quoteCurrencyKey,
-	showFee = true,
-	attached,
-	totalFeeRate,
-	baseFeeRate,
-	transactionFee,
-	feeCost,
-	isApproved = true,
-	...rest
-}) => {
-	const { t } = useTranslation();
-	const { useEthGasPriceQuery } = useSynthetixQueries();
-
-	const isSubmissionDisabled = useMemo(() => submissionDisabledReason != null, [
+const TradeSummaryCard: FC<TradeSummaryCardProps> = React.memo(
+	({
 		submissionDisabledReason,
-	]);
+		onSubmit,
+		feeReclaimPeriodInSeconds,
+		quoteCurrencyKey,
+		showFee = true,
+		totalFeeRate,
+		baseFeeRate,
+		transactionFee,
+		feeCost,
+		isApproved = true,
+		...rest
+	}) => {
+		const { t } = useTranslation();
+		const { useEthGasPriceQuery } = useSynthetixQueries();
 
-	const ethGasPriceQuery = useEthGasPriceQuery();
+		const isSubmissionDisabled = useMemo(() => submissionDisabledReason != null, [
+			submissionDisabledReason,
+		]);
 
-	const gasPrices = useMemo(
-		() => (ethGasPriceQuery.isSuccess ? ethGasPriceQuery?.data ?? undefined : undefined),
-		[ethGasPriceQuery.isSuccess, ethGasPriceQuery.data]
-	);
+		const ethGasPriceQuery = useEthGasPriceQuery();
 
-	const summaryItems = (
-		<SummaryItems attached={attached}>
-			<GasPriceSelect gasPrices={gasPrices} transactionFee={transactionFee} />
-			{showFee && (
-				<>
-					<FeeRateSummaryItem totalFeeRate={totalFeeRate} baseFeeRate={baseFeeRate} />
-					<FeeCostSummaryItem feeCost={feeCost} />
-				</>
-			)}
-		</SummaryItems>
-	);
+		const gasPrices = useMemo(() => ethGasPriceQuery?.data ?? undefined, [ethGasPriceQuery.data]);
```

Comment:
Would `ethGasPriceQuery?.data` just return undefined anyway if it doesn't exist, not sure we need `?? undefined`?

Reply:
Good point. I'm not sure. I'll look at the signature.